src/mpyl/project.py

```diff
@@ -321,9 +321,13 @@ class Kubernetes:
     command: Optional[TargetProperty[str]]
     args: Optional[TargetProperty[str]]
     labels: Optional[list[KeyValueProperty]]
+    strategy_type: str
```

Comment:
https://mypy.readthedocs.io/en/stable/literal_types.html#parameterizing-literals or an enum?

Reply:
Addressed in https://github.com/Vandebron/mpyl/pull/356/commits/69e15036831fbe7c27b8f0e4f1c03a77a26522a4